src/main/java/com/iemr/admin/service/telemedicine/VideoConsultationAPIServiceImpl.java

```diff
@@ -32,34 +32,33 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 import com.iemr.admin.utils.config.ConfigProperties;
-import com.iemr.admin.utils.exception.SwyMedException;
+import com.iemr.admin.utils.exception.VideoConsultationException;
 import com.iemr.admin.utils.http.HttpUtils;
 import com.iemr.admin.utils.mapper.InputMapper;
 import com.iemr.admin.utils.response.OutputResponse;
 
 @Service
-public class SwymedAPIServiceImpl implements SwymedAPIInter {
+public class VideoConsultationAPIServiceImpl implements VideoConsultationAPIInter {
```

Comment:
@Vidyaaa24 Change this class name to VideoConsultationServiceImpl

Reply:
ok